sway-ir/src/printer.rs

```diff
@@ -361,8 +363,10 @@ fn instruction_to_doc<'a>(
                 .append(maybe_constant_to_doc(context, md_namer, namer, asset_id))
                 .append(maybe_constant_to_doc(context, md_namer, namer, gas))
                 .append(Doc::text_line(format!(
-                    "{} = contract_call {}, {}, {}, {}{}",
+                    "{} = contract_call {} {} {}, {}, {}, {}{}",
                     namer.name(context, ins_value),
```

Comment:
Missing commas, doesn't really matter, it's a bit arbitrary.  Maybe we should take commas out of the IR altogether, it probably doesn't help readability or parsing...

Reply:
I'll open an issue and we can do some bikeshedding.